app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/218758823-4c700f1e-4cd0-4c58-ac76-8abbd7299da8.png)
fix this

Reply:
Fixed